admin_manual/configuration/user/user_types.rst

```diff
@@ -0,0 +1,80 @@
+===================
+ownCloud User Types
+===================
+
+ownCloud supports six user types.
+These are:
+
+- `Anonymous`_
+- `Guest`_
+- `Standard User`_
+- `Federated User`_
+- `ownCloud Group Administrator`_
+- `ownCloud Administrator`_
+
+Anonymous
+---------
+
+- Is not a regular user.
+- Has access to specific content via public links.
+- Password-protected (optional, enforced, policy-enforced).
+- Expiration date (optional, enforced, enforced dependent on password).
+- No personal space
+- No file ownership (ownership of uploaded/created files is directed to sharer).
+- No use of clients.
+- Using quota of sharer.
+- Permissions as granted by the sharer for specific content, e.g., view-only, edit, and no file listing.
+- Apps: Only files + viewers, such as PDF, and Collabora view/edit.
+
+Guest
+-----
+
+- Is a regular user with restricted permissions (identified via e-mail address).
+- No personal space 
+- No file ownership (ownership of uploaded/created files is directed to sharer).
+- Shared space: Permissions as granted by sharer.
+- Not bound to the 'inviting user'.
+- Can log in as long as he has shares.
+- Becomes deactivated when no shares are left ('Shared with guests' filter).
+- Reactivate when share is received.
+- Administrators will be able to automate cleanup ("disabled for x days").
+- Can use all clients.
+- Fully auditable in the enterprise edition.
+- Can be promoted to group admin/admin, but will still have no personal space.
+- Apps: As specified by the admin (whitelist).
+
+Standard User
+-------------
+
+- Is a regular user (from LDAP, ownCloud user backend or other backend).
+- Personal space: Permissions as granted by the admin.
+- Shared space: Permissions as granted by sharer.
+- Apps: All enabled, might be restricted by group membership.
+
+Federated User
+--------------
+
+- Not an internal user.
+- Federated system can be trusted.
+- Has access to shared space through users on the considered ownCloud system.
+- Can share data with the considered system (accept- / rejectable).
+
+ownCloud Group Administrator
+----------------------------
+
+- Is a regular user (from LDAP, ownCloud user backend or other backend).
+- Manage users in their groups (add/remove, change quota of users in the group).
+- Can add new users to their groups (can manage guests as well).
+- Can enable and disable users.
+- Can impersonate users in their groups.
+- Custom group creation maybe restricted to group admins.
+
+ownCloud Administrator
+----------------------
+
+- Is a regular user (from LDAP, ownCloud user backend or other backend).
+- Can configure ownCloud features via the UI (e.g., Sharing settings, app-specific configurations, external storages for users).
+- Can manages users (add/remove, enable/disable, quota, group management).
```

Comment:
Can mange users

Reply:
Or "Can manage"

Reply:
Indeed - don't want to "mange" the users :)

Reply:
heh, true.